front/src/config/config.dev.json

```diff
@@ -1,3 +1,4 @@
 {
-  "serverPath": "https://localhost:5001"
+  "baseUrl": "https://localhost:5001",
+  "apiSuffix": "/api"
```

Comment:
nit - just "api" no backslash

expressing that something could be better, but if it’s not very important, prefix it with something like “Nit: “ to let the author know that it’s just a point of polish that they could choose to ignore. source: https://google.github.io/eng-practices/review/reviewer/standard.html